modules/auxiliary/gather/piwigo_cve_2023_26876.rb

```diff
@@ -0,0 +1,168 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::SQLi
+  require 'metasploit/framework/hashes'
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Piwigo CVE-2023-26876 Gather Credentials via SQL Injection ',
+        'Description' => %q{
+          This module allows an authenticated user to retrieve the usernames and encrypted passwords of other users in Piwigo through SQL injection using the (filter_user_id) parameter.
+        },
+        'Author' => [
+          'rodnt'
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          [ 'CVE', '2023-26876' ],
+          ['URL', 'https://nvd.nist.gov/vuln/detail/CVE-2023-26876'],
+          ['Discover', 'Rodolfo Tavares'],
+          ['Thanks', 'Tempest Security', 'Henrique Arcoverde']
+
+        ],
+        'DisclosureDate' => '2023-04-21',
+        'Notes' => {
+          'Stability' => ['CRASH_SAFE'],
```

Comment:
```suggestion
          'Stability' => ['CRASH_SAFE'],
          'Reliability' = > [ ],
```
This one is on me; I thought the reliability notes was optional, but you have to have it to pass linting.  Sorry!